pyproject.toml

```diff
@@ -59,3 +59,97 @@ build-backend = "poetry_dynamic_versioning.backend"
+  "PT014",   # pytest-duplicate-parametrize-test-cases.
+  "PT006",   # Checks for the type of parameter names passed to pytest.mark.parametrize.
+  "PT007",   # Checks for the type of parameter values passed to pytest.mark.parametrize.
+  "PT018",   # Checks for assertions that combine multiple independent condition
+  "RUF001", # Checks for non unicode string literals
+  "RUF002", # Checks for non unicode string literals
+  "RUF003", # Checks for non unicode string literals
+]
+extend-select = [
+  "I", # isort
+  "C4", # https://pypi.org/project/flake8-comprehensions
+]
+ignore=[
+  "E203", # Whitespace-before-punctuation.
+  "E402", # Module-import-not-at-top-of-file.
+  "E731", # Do not assign a lambda expression, use a def.
+  "RET504", # Unnecessary variable assignment before `return` statement.
+  "S101", # Use of `assert` detected.
+  "RUF100", # https://docs.astral.sh/ruff/rules/unused-noqa/
+  "C408", # Unnecessary dict call - rewrite as a literal.
+  "UP031", # Use format specifier instead of %
+  "S102", # Use of excec
+  "C414", # Unnecessary `list` call within `sorted()`
+  "S301", # pickle and modules that wrap it can be unsafe
+  "C416", # Unnecessary list comprehension - rewrite as a generator
+  "S310", # Audit URL open for permitted schemes
+  "S202", # Uses of `tarfile.extractall()`
+  "S307", # Use of possibly insecure function
+  "C417", # Unnecessary `map` usage (rewrite using a generator expression)
+  "S605", # Starting a process with a shell, possible injection detected
+  "E741", # Ambiguous variable name
+  "S107", # Possible hardcoded password
+  "S105", # Possible hardcoded password
+  "PT018", # Checks for assertions that combine multiple independent condition
+  "S602", # sub process call with shell=True unsafe
+  "C419", # Unnecessary list comprehension, some are flagged yet are not
+  "C409", # Unnecessary `list` literal passed to `tuple()` (rewrite as a `tuple` literal)
+  "S113", # Probable use of httpx call without timeout
+]
+allowed-confusables=["σ"]
+
+[tool.ruff.lint.per-file-ignores]
+
+"setup.py" = ["S101"]
+"**/__init__.py" = [
+  "F401", # unused import
+]
+"**/tests/**" = [
+  "D",
+  "S605",  # Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`
+  "S607",  # Starting a process with a partial executable path
+  "RET504",  # todo:Unnecessary variable assignment before `return` statement
+  "PT004",  # Fixture `tmpdir_unittest_fixture` does not return anything, add leading underscore
+  "PT011",  # `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception
+  "PT012",  # `pytest.raises()` block should contain a single simple statement
+  "PT019",  # Fixture `_` without value is injected as parameter, use `@pytest.mark.usefixtures` instead
+  "PT006" # Checks for the type of parameter names passed to pytest.mark.parametrize.
+]
+"sktime/libs/uni2ts/*.py" = [
```

Comment:
@fkiraly should we remove or change this reference to `sktime`?

Reply:
yes, removed